### PR TITLE
Localization for shared/header.template

### PR DESCRIPTION
**Pull request by polarikus**
Added the ability to select the language of the documentation header, as well as Russian header.template

### CODE REVIEW

**Review by Mohammad-Alavi — APPROVED** [review]

**Review by Mohammad-Alavi — COMMENTED** [review]
What do you think if we change `header.template.md` to `header.template.en.md`? And set the default locale to `en`?

**Comment by polarikus** [comment]
Good! I think so, it will be clearer

File: Tasks/RenderTemplatesTask.php [hunk 1]
```diff
@@ -15,7 +15,7 @@ class RenderTemplatesTask extends Task
 
     public function __construct()
     {
-        $this->templatePath = 'Containers/' . config('vendor-documentation.section_name') . '/Documentation/ApiDocJs/shared/header.template.md';
+        $this->templatePath = 'Containers/' . config('vendor-documentation.section_name') . '/Documentation/ApiDocJs/shared/header.template'.config('vendor-documentation.locale', '').'.md';
```
Comment (Mohammad-Alavi) [inline Tasks/RenderTemplatesTask.php]: Please change the config to use the `en` header by default
`config('vendor-documentation.locale', 'en')`
and also change the default header to `header.template.en.md`

Comment (polarikus) [reply Tasks/RenderTemplatesTask.php]: Done! :)

File: Configs/vendor-documentation.php [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+     /*
+    |--------------------------------------------------------------------------
+    | Locale
+    |--------------------------------------------------------------------------
+    |
+    | Language header docs
+    |
+    */
+    
+    'locale' => '.'.env('APIDOC_LOCALE'),
```
Comment (Mohammad-Alavi) [inline Configs/vendor-documentation.php]: Change the `env('APIDOC_LOCALE')` to `env('APIDOC_LOCALE', 'en')`  in case there is no `env` value is set.

Comment (polarikus) [reply Configs/vendor-documentation.php]: Done! f6b73ca8f9d7ddbcf55cab91da6f03662dcb0ae9

**Pull request merged**